use of undeclared identifier 'KBUILD_MODNAME'

```
# bpftrace new.bt
[...]
/lib/modules/4.15.0-50-generic/build/include/net/inet_connection_sock.h:251:3: error: use of undeclared identifier 'KBUILD_MODNAME'
```

Some header files need this, and with recent bpftrace changes it aborts without it. I'll submit a PR to fix it.

--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -286,6 +286,7 @@ std::vector<std::string> get_kernel_cflags(
   cflags.push_back("-D__HAVE_BUILTIN_BSWAP16__");
   cflags.push_back("-D__HAVE_BUILTIN_BSWAP32__");
   cflags.push_back("-D__HAVE_BUILTIN_BSWAP64__");
+  cflags.push_back("-DKBUILD_MODNAME='\"bpftrace\"'");
 
   // If ARCH env variable is set, pass this along.
   if (archenv)

